variables.yaml.example

```diff
@@ -1,39 +1,44 @@
+google:
+  # The environment you would like to provision e.g. dev
+  # This should be the full project id if it is an existing project.
+  project_name: ''
+  folder_id: ''
+  billing_account: ''
+  region: 'europe-west2'
+  machine_type: 'n1-standard-2'
+  # The label for this project in GCP
+  project_label: 'sandbox-eq'
+
 # Currently the docker registry can only be a GCR registry.
 # Ensure in the GCR settings, the visibility is set to Public.
-# The gcp_project_id should be the project id where the container registry is hosted.
+# The google.project_id should be the project id where the container registry is hosted.
 # This is the docker registry to pull the application images from.
-app_docker_registry: eu.gcr.io/<gcp_project_id}>
-
-# The docker registry to pull the build images from.
-build_image_docker_registry: eu.gcr.io/census-eq-ci
-
-import_existing_project: false
-# The environment you would like to provision e.g. dev
-# This should be the full project id if it is an existing project.
-project_name: ''
+docker:
+    app_registry: eu.gcr.io/<google.project_id>
+    # The docker registry to pull the build images from.
+    build_image_registry: eu.gcr.io/census-eq-ci
 
 # If a non development state bucket for the terraform backend is used, specify it here
-load_generator_tf_state_bucket: 'eq-terraform-load-generator-tfstate'
+terraform_state_bucket: 'eq-terraform-load-generator-tfstate'
 
 # Override the default kubernetes version if necessary
 k8s_min_master_version: 1.13
-gcp_folder_id: ''
-gcp_billing_account: ''
-region: 'europe-west2'
 
 slack_channel_name: ''
 
-# The label for this project in GCP
-gcp_project_label: 'sandbox-eq'
-
 # The full DNS name for runner
 # For example: myenv-runner.gcp.dev.eq.ons.digital
 runner_fully_qualified_domain_name: ''
 
 requested_cpu_per_pod: '0.7'
 requested_memory_per_pod: '2048Mi'
+
+# How many instances of the test runner to use
 parallelism: 1
 user_wait_time_min_seconds: 0
 user_wait_time_max_seconds: 0
 requests_json: 'requests/census_individual_gb_eng.json'
-locust_options: '--no-web -c 1 -r 1 -t 10m --csv=output -L WARNING'
+number_of_clients_per_pod: 10
+hatch_rate: 10
+run_time: '30m'
+timeout: '60m'
```

Comment:
I think it's worth considering structuring the variables file to align with the vars in the pipeline. For example we could have some top level properties for `google`, `infrastructure` and `benchmark`. Anything that applies to more than one thing can be left at the top level.